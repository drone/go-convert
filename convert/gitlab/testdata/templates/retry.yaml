test:
  script: rspec
  retry: 2