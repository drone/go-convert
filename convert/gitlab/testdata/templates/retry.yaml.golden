kind: pipeline
spec:
  stages:
  - name: test
    spec:
      steps:
      - failure:
          action:
            spec:
              attempts: 2
            type: retry
        name: test
        spec:
          run: rspec
        type: script
    type: ci
version: 1
